tfhe-benchmark/benches/integer/vector_find.rs

```diff
@@ -0,0 +1,217 @@
+use benchmark::params_aliases::BENCH_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128;
+use benchmark::utilities::{write_to_json, OperatorType};
+use criterion::{black_box, criterion_group, Criterion};
+use tfhe::integer::keycache::KEY_CACHE;
+use tfhe::integer::{IntegerKeyKind, RadixClientKey};
+use tfhe::keycache::NamedParam;
+use tfhe::shortint::AtomicPatternParameters;
+use tfhe::MatchValues;
+
+pub fn match_value(c: &mut Criterion) {
+    let bench_name = "integer::match_value";
+
+    let param = BENCH_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128;
+    let atomic_param: AtomicPatternParameters = param.into();
+    let param_name = param.name();
+
+    let (cpu_cks, cpu_sks) = KEY_CACHE.get_from_params(atomic_param, IntegerKeyKind::Radix);
+
+    //
+    // Latency test: 64 bits, 10 elements
+    //
+    {
+        let num_elements = 10;
+        let bits = 64;
+        let num_blocks = 32;
+
+        let bench_id =
+            format!("{bench_name}::{param_name}::latency_{bits}bit_{num_elements}elements");
+
+        let cks = RadixClientKey::from((cpu_cks.clone(), num_blocks));
+
+        let mapping_data: Vec<(u64, u64)> = (0..num_elements as u64).map(|i| (i, i)).collect();
+        let match_values = MatchValues::new(mapping_data).expect("Invalid match values");
+
+        let input_val = 1u64;
+        let ct_input = cks.encrypt(input_val);
+
+        let mut group = c.benchmark_group(&bench_id);
+        group.sample_size(15);
+
+        group.bench_function("latency", |b| {
```

Comment:
Prefer the `bench_id` as a identifier rather than `"latency"`, even if criterion will most likely truncate the `bench_id` string.